src/content/guides/build-performance.md

```diff
@@ -109,6 +109,10 @@ Enable persistent caching with the `cache-loader`. Clear cache directory on `"po
 
 Profile them to not introduce a performance problem here.
 
+### Progress plugin
+
+If you already have a fast build time, `progress-plugin` can be excessive. If you have long build times, consider removing `progress-plugin` where the build happens to be remotely, for example your server or CI.
```

Comment:
How does that read to you guys?
```suggestion
It is possible to shorten build times by removing `progress-plugin` from webpack's configuration. Keep in mind, `progress-plugin` might not provide as much value for fast builds as well, so make sure you are leveraging the benefits of using it.
```

Reply:
Sounds great. I am going to change it to your suggestion. Though I think it is worth to note the disadvantage of having `progress-plugin` extensively populating logs.

Reply:
Its not directly related to the build . prerformance guide, how about opening a pull request to update progress-plugin page with that warning? (logs size)

Reply:
Great idea. Going to do it once this PR is merged. Thank you!